src/librustc_codegen_llvm/lib.rs

```diff
@@ -39,13 +39,14 @@ extern crate rustc_incremental;
 extern crate rustc_codegen_utils;
 extern crate rustc_codegen_ssa;
 extern crate rustc_fs_util;
+#[allow(unused_extern_crates)]
+extern crate rustc_driver;
```

Comment:
`unused_extern_crates` can also be silenced with `extern crate rustc_driver as _;`